test/integration/forms/form.js

```diff
@@ -145,96 +166,186 @@ context('Noncontext Form', function() {
       })
-      .routePatient(fx => {
-        fx.data.id = '1';
-        return fx;
-      })
       .routeForm(_.identity, '11111')
       .routeFormFields()
-      .visit('/patient/1/form/11111')
+      .visit(`/patient/${ patientId }/form/11111`)
       .wait('@routePatient')
       .wait('@routeForm')
       .wait('@routeFormFields')
       .wait('@routeFormDefinition');
 
+    // NOTE: .wait(100) account for form.io data delays
+
     cy
-      .wait('@routePatchPatientFieldFoo')
-      .wait('@routePatchPatientFieldBar')
-      .its('request.body.data.attributes')
-      .should('deep.equal', { name: 'bar', value: ['bar', 'baz'] });
+      .iframe()
+      .find('button')
+      .contains('Test Get Field')
+      .click()
+      .wait('@routePatientFieldfoo')
+      .wait(100);
 
     cy
       .iframe()
       .find('.formio-component-select .dropdown')
+      .click();
+
+    cy
+      .iframe()
+      .find('.choices__list--dropdown.is-active')
+      .find('.choices__item--selectable')
+      .should('contain', '1')
+      .should('contain', '2')
       .first()
       .click();
 
+    cy
+      .iframe()
+      .find('button')
+      .contains('Test Get Error')
+      .click()
+      .wait('@routePatientFieldbar')
+      .wait(100);
+
+    cy
+      .iframe()
+      .find('.formio-component-select .dropdown')
+      .click();
+
     cy
       .iframe()
       .find('.choices__list--dropdown.is-active')
       .find('.choices__item--selectable')
+      .should('contain', 'error')
       .first()
+      .click();
+
+    cy
+      .iframe()
+      .find('button')
+      .contains('Test Update Field')
+      .click()
+      .wait('@routePatchPatientFieldFoo')
+      .its('request.body.data')
+      .then(data => {
+        expect(data.id).to.equal('1');
+        expect(data.attributes.name).to.equal('foo');
+        expect(data.attributes.value).to.deep.equal(['one', 'two']);
+      })
+      .wait(100);
+
+    cy
+      .iframe()
+      .find('.formio-component-select .dropdown')
+      .click();
+
+    cy
+      .iframe()
+      .find('.choices__list--dropdown.is-active')
+      .find('.choices__item--selectable')
       .should('contain', 'one')
-      .next()
       .should('contain', 'two')
+      .first()
       .click();
 
+    cy
+      .iframe()
+      .find('button')
+      .contains('Test Update Error')
+      .click()
+      .wait('@routePatchPatientFieldBar')
+      .its('request.body.data.id')
+      .should('equal', uuid('resource:field:bar', patientId))
+      .wait(100);
+
     cy
       .iframe()
       .find('.formio-component-select .dropdown')
-      .last()
       .click();
 
     cy
       .iframe()
       .find('.choices__list--dropdown.is-active')
       .find('.choices__item--selectable')
+      .should('contain', 'error')
       .first()
-      .should('contain', 'Error: Bad Request');
+      .click();
   });
 
   specify('form scripts and reducers', { retries: 4 }, function() {
     cy
+      .route({
```

Comment:
this tests a form load check, but isn't worth it's own specify.  I moved it to a more generic test than the directory one